client/app.jsx

```diff
@@ -29,17 +31,24 @@ class App extends React.Component {
       });
   }
 
-
-
   render() {
-    const images = this.state.imageUrls.map(imageUrl =>
-      <img src={imageUrl} alt="shirt"  width="400" />
-    );
-      console.log(this.state.imageUrls)
+    const images = this.state.imageUrls.map(imageUrl => {
+      return (
+      <div className='imgDiv'>
+        <img src={imageUrl} alt="shirt" />
+      </div>
+      )
+    });
+
+    const settings = {
+      dots: false
```

Comment:
Can you change dots to something else, just for clearer variable naming?

Reply:
"dots" is a prop name used by the react-slick library.